auditbeat/module/file_integrity/config.go

```diff
@@ -72,6 +74,20 @@ const (
 	XXH64       HashType = "xxh64"
 )
 
+type Backend string
+
+const (
+	BackendFSNotify Backend = "fsnotify"
+	BackendKprobes  Backend = "kprobes"
+	BackendEBPF     Backend = "ebpf"
+	BackendAuto     Backend = "auto"
+)
+
+func (b *Backend) Unpack(v string) error {
+	*b = Backend(v)
```

Comment:
I recommend applying some validation at this point to ensure that the given value is a valid backend type. Catching an invalid value during the "unpacking" phase will allow `auditbeat test config` to give more accurate feedback.